### PR TITLE
[alertmanager] switches to global wham url

### PR DESCRIPTION
**Pull request by stefanhipfel**
- activates canary regions for alert bm error handling

### CODE REVIEW

**Comment by auhlig** [comment]
`continue: false` := final receiver if regex' match.
Critical and warning alerts will never reach wham, since they are caught here: https://github.com/sapcc/helm-charts/blob/d50f48bcf60db46c79e5cb0c791c2f4603f0532f/global/prometheus-alertmanager/templates/config.yaml#L158-L170.

**Comment by stefanhipfel** [comment]
So slack_metal_critical and slack_metal_warning should enable continue. 
Do we allow to set it to true? :)

File: global/prometheus-alertmanager/templates/config.yaml [hunk 1]
```diff
@@ -251,7 +251,7 @@ data:
 
       # ---- webhook handler: metal ----
       - receiver: wham_metal
-        continue: false
+        continue: true
```
Comment (auhlig) [inline global/prometheus-alertmanager/templates/config.yaml]: being the last receiver for metal alerts this is `continue: false`

Comment (stefanhipfel) [reply global/prometheus-alertmanager/templates/config.yaml]: ah ok, should it be last one? (or is that mandatory to have a last handler?)...i saw my handler more like a "middleware"..

Comment (auhlig) [reply global/prometheus-alertmanager/templates/config.yaml]: in this case wham is the last receiver. the [dev-null receiver](https://github.com/sapcc/helm-charts/blob/776e03ead9d67d96a6ca5b957cb4d1f3df3e5dbb/global/prometheus-alertmanager/templates/config.yaml#L260-L262) is meant to catch alerts that aren't labeled properly. for this work properly labeled alerts need to have a final receiver before

Comment (stefanhipfel) [reply global/prometheus-alertmanager/templates/config.yaml]: ok got it!

**Pull request merged**